lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -0,0 +1,79 @@
+module Fastlane
+  module Actions
+    class AndroidDownloadFileByVersionAction < Action
+      def self.run(params)
+        require_relative '../../helper/android/android_localize_helper.rb'
+        require_relative '../../helper/github_helper'
+
+        version = Fastlane::Helper::Android::LocalizeHelper.get_library_version_from_gradle_config(import_key: params[:import_key])
```

Comment:
Just realized that this namespace/module doesn't seem like the best place to host such a method? (not sure how `get_library_version_from_gradle_config` needs to be specific to Localization helpers)… what about moving it to `android_version_helper` (since it's dealing with parsing versions from gradle file, which this helper already provide similar methods about)?